vimeo-networking/src/main/java/com/vimeo/networking/model/Video.java

```diff
@@ -102,17 +105,25 @@ public String toString() {
     public Date modifiedTime;
     public ArrayList<String> contentRating;
     public String license;
-    public com.vimeo.networking.model.Privacy privacy;
+    public Privacy privacy;
     public PictureCollection pictures;
     public ArrayList<Tag> tags;
     public StatsCollection stats;
     public Metadata metadata;
-    public com.vimeo.networking.model.User user;
+    public User user;
     private Status status;
     public VideoLog log;
     public List<Category> categories;
 
+    /**
+     * This getter is always guaranteed to return a {@link Status}, {@link Status#NONE if null}. If the Status
+     * is equal to {@link Status#TRANSCODE_STARTING} then we'll just return the Status {@link Status#TRANSCODING}
+     * since they're functionally equivalent from a client-side perspective.
+     */
     public Status getStatus() {
+        if (status == Status.TRANSCODE_STARTING) {
+            return Status.TRANSCODING;
+        }
```

Comment:
I know you and @alfiehanssen discussed this, but I am not too sure this is the correct way to go. Perhaps other consumers would like to know when we are in the transcode starting state, if we go with this then they wouldn't be able to account for it. Likewise, we may want to be privy to that status in the future as well. From our UI side it is currently equivalent, but this doesn't reflect a decision from our API team to separate the two.